app/views/contexts/calculated_summary_context.py

```diff
@@ -116,24 +128,24 @@ def build_view_context(self) -> dict[str, dict]:
         )
 
         return self._build_formatted_summary(
-            block=block,
             groups=groups,
             calculation=calculation,
             formatted_total=formatted_total,
             summary_type="CalculatedSummary",
+            block_type="calculated-summary",
         )
 
     def _build_formatted_summary(
         self,
         *,
-        block: ImmutableDict,
         groups: Iterable[Mapping],
-        calculation: ImmutableDict,
+        calculation: Mapping,
         formatted_total: str,
         summary_type: str,
+        block_type: str,
```

Comment:
Can we not remove both these types and instead use `self.rendered_block["type"]`, `yes block type is hyphen separated but could be handled dynamically instead of passing them in?.

Reply:
Sure - made it a separate string method so it can always be picked up elsewhere if needed